src/qt/pivx/send.cpp

```diff
@@ -751,6 +763,13 @@ void SendWidget::onCheckBoxChanged()
 void SendWidget::onPIVSelected(bool _isTransparent)
 {
     isTransparent = _isTransparent;
+
+    if (!isTransparent && !walletModel->isSaplingEnforced()) {
+        ui->pushLeft->setChecked(true);
+        inform(tr("Cannot perform shielded operations, v5 upgrade hasn't being enforced yet!"));
```

Comment:
change to either `hasn't been enforced` or `isn't being enforced`

Reply:
typo fixed